app/api/v2/utils.py

```diff
@@ -20,7 +20,30 @@ def doPasswordsMatch(pass1, pass2):
     """
     if(pass1 != pass2):
         abort(response_fn(400, "error", "passwords dont match"))
-    return True
+    check_password_format(pass1)
+
+
+def check_password_format(password):
+    '''
+     Does password meet expectations
+     of passwords that need to be saved
+    '''
+    # check to confirm the password is of required length
+    if len(password) < 5 or len(password) > 20:
+        abort(response_fn(400, "error",
+                          "Password should not be less than 8 characters or exceed 20"))
+
+    # check if password contains at least an alphabet(a-z)
+    if not re.search("[a-z]", password):
+        abort(response_fn(400, "error", "Password should contain a letter between a-z"))
+
+    # check if password contains at least an upper case letter
+    if not re.search("[A-Z]", password):
+        abort(response_fn(400, "error", "Password should contain a capital letter"))
+
+    # check if password contains at least a number(0-9)
+    if not re.search("[0-9]", password):
+        abort(response_fn(400, "error", "Password should contain a number(0-9)"))
```

Comment:
You should combine these regexes into one, and give a general message that password should contain an uppercase, lowercase and a number

Reply:
Okay, implementing that in a bit